pkg/apis/provisioning/v0alpha1/jobs.go

```diff
@@ -94,15 +100,23 @@ type ExportJobOptions struct {
 	// The source folder (or empty) to export
 	Folder string `json:"folder,omitempty"`
 
-	// Preserve history (if possible)
-	History bool `json:"history,omitempty"`
```

Comment:
removing history from "export" for now -- since we only support it from "migrate"